### PR TITLE
WordPress Post GUIDs: Sometimes You Should Update Them

### PR DESCRIPTION
**Pull request by bradt**
It's a little short. Any suggestions for diving into more things?

### CODE REVIEW

**Comment by bradt** [comment]
And btw, you might want to use the "rich diff" feature when reviewing changes:
https://github.com/deliciousbrains/blog-drafts/commit/d41500016c6a1a0a62c0b2d42125e8a6350f50de?short_path=4991776#diff-499177675d259c4e524486f43323f650

File: 20150228-wordpress-guids.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+#### How does WordPress use GUIDs?
+
+WordPress employs GUIDs when it generates its RSS (and Atom) feeds. If you take a look at the [RSS feed of this blog](https://deliciousbrains.com/blog/feed/) you will see elements similar to this:
+
+```
+<guid isPermaLink="false">https://deliciousbrains.com/?p=6806</guid>
+```
+
+WordPress uses a real, working post URL as the GUID. In fact, if you go to the URL in the example above, it will redirect you to the blog post it belong to. This is also the URL the post has when using the default Permalink setting. Since URLs are globally unique by their nature, they make pretty good GUIDs so long as they don't change. And in this case, since we're using the post ID in the URL, it's almost certain not to change. A good design decision.
```
Comment (A5hleyRich) [inline 20150228-wordpress-guids.md]: redirect you to the blog post it belong**s** to

File: 20150228-wordpress-guids.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+#### What are GUIDs used for in RSS feeds?
+
+RSS feed readers (like Feedly) use GUIDs to uniquely identify an item against all other items that have appeared in the feed and will appear in the feed in the future. It uses the GUID to determine if it should show an item as new.
```
Comment (A5hleyRich) [inline 20150228-wordpress-guids.md]: the feed and will appear **in the feed** in the future

Is this necessary? Feed overload.

**Comment by A5hleyRich** [comment]
> It's a little short. Any suggestions for diving into more things?

You've covered everything that I had in mind and it's a pretty short topic, so you'll be pressed to add more without it sounding monotonous. I think it's just right at the minute - a short read while still being informative.

**Comment by polevaultweb** [comment]
Looks good to me, and agree with @A5hleyRich, succinct and says what it needs to. Also probably quite refreshing in length considering the last two posts have been long ones!

File: 20150228-wordpress-guids.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#### How does WordPress use GUIDs?
+
+WordPress employs GUIDs when it generates its RSS (and Atom) feeds. If you take a look at the [RSS feed of this blog](https://deliciousbrains.com/blog/feed/) you will see elements similar to this:
+
+```
+<guid isPermaLink="false">https://deliciousbrains.com/?p=6806</guid>
+```
+
+WordPress uses a real, working post URL as the GUID. In fact, if you go to the URL in the example above, it will redirect you to the blog post it belongs to. This is also the URL the post has when using the default Permalink setting. Since URLs are globally unique by their nature, they make pretty good GUIDs so long as they don't change. And in this case, since we're using the post ID in the URL, it's almost certain not to change. A good design decision.
```
Comment (ianmjones) [inline 20150228-wordpress-guids.md:29]: There's a possible opportunity to reference the post about not merging databases here, based on the fact that a merge might change the ID of a post when the ID has been used in both databases.

**Comment by ianmjones** [comment]
Great piece Brad, and as the others have said it's plenty long enough as it covers the subject nicely and people will more likely read and link to it now rather than put it off for later (I have a massive Instapaper queue of great articles that I'm less likely to share as I get through them as their time has effectively come and gone).

File: 20150228-wordpress-guids.md (hunk 3 above)
Comment (bradt) [inline 20150228-wordpress-guids.md:29]: I think that would be a bit a of digression though. I'd like to stay focused on the problem at hand and not delve into another problem.

**Comment by bradt** [comment]
Good points Ian, I hadn't really considered short is better for sharing.

**Pull request merged**